Src/Orbiter/FlightRecorder.cpp

```diff
@@ -316,7 +316,8 @@ void Vessel::FRecorder_SaveEvent (const char *event_type, const char *event)
 void Vessel::FRecorder_SaveEventInt (const char *event_type, int event)
 {
 	static char cbuf[24];
-	FRecorder_SaveEvent (event_type, _itoa (event, cbuf, 10));
+	sprintf(cbuf, "%d", event);
+	FRecorder_SaveEvent (event_type, cbuf);
```

Comment:
```c++
FRecorder_SaveEvent(event_type, std::to_string(event).data());
```
And drop `static char cbuf[24];` altogether.

Reply:
Or better yet, update the `Vessel` functions, so we don't have to come back to it again:
```diff
-	void FRecorder_SaveEvent (const char *event_type, const char *event);
+	void FRecorder_SaveEvent(std::string_view event_type, std::string_view event);

-	void FRecorder_SaveEventInt (const char *event_type, int event);
+	void FRecorder_SaveEvent(std::string_view event_type, int event);

-	void FRecorder_SaveEventFloat (const char *event_type, double event);
+	void FRecorder_SaveEvent(std::string_view event_type, double event);
```

Reply:
OK for the first part, but I won't change the API for now